types/artifacts.d.ts

```diff
@@ -239,14 +239,8 @@ declare global {
           left: number;
           right: number;
         };
-        networkRecord?: {
-          url: string;
-          resourceSize: number;
-          startTime: number;
-          endTime: number;
-          responseReceivedTime: number;
-          mimeType: string;
-        };
+        resourceSize?: number;
```

Comment:
this default's to 0 so no need to be optional?

Reply:
done